src/views/developers/l10n.json

```diff
@@ -32,13 +40,19 @@
   "developers.designPrinciplesGlobalBody": "Many math and science activities have traditionally been biased towards specific populations. By paying special attention to creating accessible and appealing technologies, we are working to close the gap.",
   "developers.designPrinciplesTinkerTitle": "Design for Tinkerability",
   "developers.designPrinciplesTinkerBody": "We believe that the learning process is inherently iterative. Tinkerers start by exploring and experimenting, then revising and refining their goals and creations. To support this style of interaction, we design our interfaces to encourage quick experimentation and rapid cycles of iteration.",
-  "developers.joinBody": "We're a diverse group of educators, designers, and engineers, who work together in a playful, creative environment full of LEGO bricks, craft materials, and maker tools. We strongly value diversity, collaboration, and respect in the workplace. If you're interested in joining us, take a look at our open positions on our <a href=\"/jobs\">Jobs Page</a>, or send us an email at <a href=\"mailto:jobs+developers@scratch.mit.edu\">jobs+developers@scratch.mit.edu</a>.",
-  "developers.donateIntro": "We are pleased to provide Scratch free of charge. If you enjoy using Scratch, please consider <a href=\"https://secure.donationpay.org/scratchfoundation/\">making a donation to support Scratch</a>. Donations of any size are appreciated.",
+  "developers.joinBodyJobsLinkText": "Jobs Page",
+  "developers.joinBodyEmailLinkText": "jobs+developers@scratch.mit.edu",
```

Comment:
This one

Reply:
The email doesn't need to be translated.